src/lib/input/input.md

```diff
@@ -150,3 +150,6 @@ Here are the available global options:
 | Name              | Type     | Description |
 | ----------------- | -------- | ----------- |
 | errorStateMatcher | Function | Returns a boolean specifying if the error should be shown |
+
+### Accessibility
+The `mdInput` directive works with native `<input>` to provide an accessible experience.
```

Comment:
Should also mention that if you add a `placeholder` attribute to the input or a `md-placeholder` element in the form field that placeholder text will automatically be used as the label for the input. If you don't specify any placeholder you need to add `aria-label` `aria-labeledby` or `<label for=...>`

Possibly also worth mentioning that any `md-error` and `md-hint` are automatically added to the input's `aria-describedby`